testsuite/run_sets/testsuite.yml

```diff
@@ -10,6 +10,7 @@
 
 - features/core_first_settings.feature
 # initialize SUSE Manager server
+- features/core_srv_menu.feature
```

Comment:
To be made secondary, and merged with the other feature that does also check menus.

Reply:
Addressed in https://github.com/uyuni-project/uyuni/pull/937/commits/dad89c163b875c03668ca18b273e96321ad88792